etl/steps/data/garden/regions/2023-01-01/regions.yml

```diff
@@ -474,9 +474,9 @@
     - "DEU"
 
 - code: "TLS"
-  name: "Timor"
+  name: "East Timor"
   aliases:
-    - "East Timor"
+    - "Timor"
```

Comment:
I think we shouldn't have this alias. Timor is an island that includes East Timor and a part of Indonesia.